tests/components/LocationBias.test.tsx

```diff
@@ -0,0 +1,148 @@
+import {fireEvent, render, screen, waitFor} from "@testing-library/react";
```

Comment:
Could you run eslint and fix any errors? Probably all of them can be autofixed with the `--autofix` flag. 
https://eslint.org/docs/user-guide/command-line-interface

Reply:
done!